example/synchronous_resnet/resnet_controller.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import absolute_import, print_function
+import os
+import sys
+import time
+
+import numpy
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+from platoon.channel import Controller
+
+
+class ResNetController(Controller):
```

Comment:
I mostly copied everything from the lstm example and work from there. You are right though that if I do a diff between the lstm_controller and resnet_controller there is not difference except the names. There could just be a general controller used by both examples. I do have to implement the method `handle_control` and subclass the general `Controler` class.